{
  "file_data": [
    {
      "desc": {
        "cause": "Under Investigation",
        "cluster": false,
        "cust_a": {
          "masked": true,
          "val": 4
        },
        "etr": "2020-11-17T15:00:00-0400",
        "start": "2020-11-17T09:18:05-0400"
      },
      "geom": {
        "a": [
          "irpoG|wccKgBcCHU|AxC"
        ],
        "p": [
          "ktpoG|tccK"
        ]
      },
      "id": "6",
      "title": "Outage Information"
    }
  ],
  "file_title": "030322"
}
